### DISCUSSION TITLE
Timestamp au lieu du texte

### DISCUSSION THREAD

**Discussion by volcanofr** [Crownicles/Crownicles#1977] [Nouvelles idées]
Remplacer les "durera environ **x unités**" par un [timestamp](https://r.3v.fi/discord-timestamps/ 'Créer un Discord timestamp') relatif.
Ca permettra de savoir combien de temps il reste à partir de l'instantanné.

> Replace these "**x hours**" to a relative [timestamp](https://r.3v.fi/discord-timestamps/ 'Create a Discord timestamp').
> With that, the text will be updated automatically by Discord for the actual time.

**Comment by voltou91** [comment]
On est d'accord que tu parles de ça ? 
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/62652451/dc3d6876-42eb-49f3-be16-04fce08872a1)

Si c'est le cas, je ne voit pas trop l'intérêt, c'est une information sur la durée totale du trajet avant ton prochain gros event. 
Contrairement à quand tu refait /report pour avoir une information sur ton prochain arrêt (mini-event) où là c'est avec les timestamp de Discord.
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/62652451/5fa47470-57ea-4a30-b726-739ccc38b969)

Je sais pas si t'as compris ce que je voulais dire.

**Comment by Nargacaura** [comment]
Je ne vois pas vraiment l'intérêt de mettre un timestamp sur une durée *théorique* et dont l'alté potentielle n'est pas comptabilisée.
Après, si c'est sur le profil, why not?

**Comment by leuhvraiminoryo** [comment]
j'trouve ça peu intéressant aussi, pour les raisons déjà évoquées. En fait, je suis même plutôt contre, même pour le coup du profil
